core/src/main/java/org/apache/carbondata/core/metadata/schema/table/column/CarbonDimension.java

```diff
@@ -39,6 +39,16 @@
    */
   private int keyOrdinal;
 
+  private byte[] childElements;
```

Comment:
This looks to be row level info. schema should not care about row level children.

please remove it

Reply:
done